### PR TITLE
React Components Insta-Clone

### PR DESCRIPTION
**Pull request by tmshkr**

### CODE REVIEW

**Comment by tmshkr** [comment]
[Live Site](https://tmshkr.github.io/React-Components-Insta-Clone/)

File: package.json [hunk 1]
```diff
@@ -9,12 +9,18 @@
...
+  "homepage": "http://tmshkr.github.io/React-Components-Insta-Clone",
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "predeploy": "react-scripts build",
+    "deploy": "gh-pages -d build"
   },
```
Comment (jskway) [inline package.json:20]: Nice work adding the gh-pages package and deploy script :100:

File: src/components/CommentSection/CommentInput.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const CommentInput = props => {
         type="text"
         value={props.comment}
         placeholder="Add comment... "
-        onChange={props.changeComment}
+        onChange={e => props.setComment(e.target.value)}
```
Comment (jskway) [inline src/components/CommentSection/CommentInput.js:11]: Nice job with the comments feature :+1:

File: src/components/PostsContainer/LikeSection.js [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
-        <i className="far fa-heart" />
-      </div>
-      <div className="like-section-wrapper">
-        <i className="far fa-comment" />
+      <div className="like-section" key="likes-icons-container">
+        <div className="like-section-wrapper">
+          <i
+            className={`${liked ? "fas" : "far"} fa-heart like-button`}
+            style={liked ? { color: "#f44336" } : null}
+            onClick={() => setLiked(!liked)}
```
Comment (jskway) [inline src/components/PostsContainer/LikeSection.js:16]: Awesome job implementing the color and like count toggling

**Review by jskway — APPROVED** [review]

**Pull request merged**